.eslintrc

```diff
@@ -262,4 +266,4 @@ rules:
   valid-typeof: [2]
   wrap-iife: [2, inside]
   yield-star-spacing: [2, {before: true, after: false}]
-  yoda: [2, never]
+  yoda: [0]
```

Comment:
In the `edit.js` file on lines 66, 67 (`"handled" == CodeMirror` ) and 83, 109 ( `"checkbox" == node.type`) - [ref](https://github.com/openstyles/stylus/blob/master/edit.js#L66-L109).